src/utils/atoms/combineAtoms.ts

```diff
@@ -13,3 +14,58 @@ export function combineAtoms<T extends AtomsMap>(
     }, {} as { [key in keyof T]: T[key] extends Atom<infer S> ? S : unknown });
   });
 }
+
+type AsyncAtomsMap = { [key: string]: AsyncAnyAtom<AtomBinded<any>> };
+type CombinedAsyncAtomsState<T> = {
+  loading: boolean;
+  error:
+    | {
+        [key in keyof T]: T[key] extends AsyncAnyAtom<infer P, infer F>
+          ? AtomState<T[key]>['error']
+          : unknown;
+      }
+    | null;
+  data: {
+    [key in keyof T]: T[key] extends AsyncAnyAtom<infer P, infer F>
+      ? AtomState<T[key]>['data']
+      : unknown;
+  };
+  lastParams: {
+    [key in keyof T]: T[key] extends AsyncAnyAtom<infer P, infer F>
+      ? AtomState<T[key]>['lastParams']
+      : unknown;
+  };
+  dirty: true;
```

Comment:
I'd like to have an explanation for `dirty`. There's no explanation for it in `createAsyncAtom` too

Reply:
hmm, it's pretty standard thing
https://en.wikipedia.org/wiki/Cache_(computing)#Dirty
Not sure how to explain in short comment